libshortfin/tests/amdgpu_system_test.py

```diff
@@ -4,7 +4,12 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+import os
+import pytest
 
+GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
+
+@pytest.mark.skipif(GITHUB_ACTIONS, reason="No AMD GPU in GitHub action.")
```

Comment:
I'm not sure if i like this. Can we instead label these tests as "requires-amdgpu" and in github actions filter these tests out?

Reply:
I don't know if we can use a label here, but let me rename the environment variable and set it explicitly.